packages/malloy/src/lang/ast/expressions/expr-func.ts

```diff
@@ -369,3 +435,34 @@ function findOverload(
     }
   }
 }
+
+type InterpolationPart =
+  | {type: 'string'; value: string}
+  | {type: 'interpolation'; name: string};
+
+function parseSQLInterpolation(template: string): InterpolationPart[] {
```

Comment:
It doesn't look like either of us added a way to include `${` literally in the SQL string. Not sure how important that is at this stage, but it might be worth doing a `.replace('\$', '$')` or something in the `{type: 'string'}` case to allow for that.

Reply:
Great call out.  I'll take a look at this today and update.  Thanks!

Reply:
Per discussion with you in chat, since the Looker parser doesn't do this, no need to do it for now.